Library/Formula/parse-cli.rb

```diff
@@ -0,0 +1,17 @@
+class ParseCli < Formula
+  homepage "https://parse.com"
+  url "https://www.parse.com/downloads/cloud_code/parse",
```

Comment:
To go further on that: if/when you resubmit please provide versioned links, a way of disabling the auto-updater (even if it's just editing a line in the file), a better test (one that does more than `--version` or `--help`) and either a way of building from source or submit it to `homebrew-binary`. Thanks!